app/js/index.js

```diff
@@ -6,6 +6,7 @@ import { I18nextProvider } from 'react-i18next';
 import i18n from './i18n';
 
 import 'bootstrap/dist/css/bootstrap.css';
+import '../css/fonts/Inter/inter.css';
```

Comment:
if you want the storybook to have the fonts too, you need to add it into `.storybook/config.js` too